src/backend/web/renderers/team_renderer.py

```diff
@@ -182,8 +184,32 @@ def render_team_details(
                     None,
                 )
 
+            alliance_size = AllianceHelper.get_known_alliance_size(
+                event.event_type_enum, event.year
+            )
+
+            alliance, alliance_pick = AllianceHelper.get_alliance_details_and_pick_name(
```

Comment:
`get_known_alliance_size` is called within this helper. Would it make sense to remove the redundant call above and have `get_alliance_details_and_pick_name` also return `alliance_size`?